Describe PieChart annotations (#1318)
DevExpress/devextreme-documentation: api-reference/_hidden/dxPieChartCommonAnnotationConfig/series.md

---
---
##### shortDescription
Anchors the annotation to a series point. Accepts the [name](/Documentation/ApiReference/Data_Visualization_Widgets/dxPieChart/Configuration/series/#name) of the point's series.

---

> Use this option when the annotation is positioned relative to an [argument](/api-reference/_hidden/BaseChartAnnotationConfig/argument.md '{basewidgetpath}/Configuration/annotations/#argument').
> 
> You can omit this option if the **PieChart** has only one [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxPieChart/Configuration/series/). 

Reviewer: ```suggestion
Use this option when the annotation is positioned relative to an [argument](/api-reference/_hidden/BaseChartAnnotationConfig/argument.md '{basewidgetpath}/Configuration/annotations/#argument') and the **PieChart** contains more than one [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxPieChart/Configuration/series/).
```

---
---
##### shortDescription
Anchors the annotation to a series point. Accepts the [name](/Documentation/ApiReference/Data_Visualization_Widgets/dxPieChart/Configuration/series/#name) of the point's series.

---

> Use this option when the annotation is positioned relative to an [argument](/api-reference/_hidden/BaseChartAnnotationConfig/argument.md '{basewidgetpath}/Configuration/annotations/#argument').

Reviewer: ...and when there are multiple series?

> For example, the following **PieChart** fragment displays two series and an annotation. The annotation is positioned relative to the argument `B` but not anchored to any of the two series points that correspond to this argument. To anchor the annotation to a series point, specify the annotation's **series** option:

Reviewer: Illustrations are needed.

Author: Seems like this part should not be included since you cannot create annotations in PieChart that are not anchored to a series point.
